xml/advanced_topics_proxy_quickstart.xml

```diff
@@ -62,6 +62,15 @@
    The primary goal of a &susemgrproxy; is to improve &susemgr; performance by
    reducing bandwidth requirements and accelerating response time.
   </para>
+
+  <note>
+   <title>Version Information</title>
+   <para>
+    By now (October 2018), SLE 12 SP4 is available and &suse; recommands
+    to use SLE 12 SP4 as the base system, either for installation or as
+    the migration target.
```

Comment:
"SLE 12 SP4 became available in October 2018. SUSE recommends that you use this as the base system for installation, or as your migration target."

(Note 'recommends' typo in original, too)

Reply:
SLE 12 SP4 is **not** yet general available!!!

FCS is somewhere mid of December.